main/src/main/java/org/mobilitydata/gtfsvalidator/validator/LevelPresenceValidator.java

```diff
@@ -68,15 +68,15 @@ public void validate(NoticeContainer noticeContainer) {
    * A row from pathways.txt has {@code pathways.pathway_mode=5} but levels.txt is empty or not
    * provided.
    *
-   * <p>Severity: {@code SeverityLevel.WARNING}. To be upgraded to {@code SeverityLevel.ERROR}.
+   * <p>Severity: {@code SeverityLevel.ERROR}.
```

Comment:
This validation rule will be fully rewritten. I do not have statistics for breaking that rule in Google feeds. I suggest to postpone this modification until we have the data.

By the way, what does your statistics say?

Reply:
> This validation rule will be fully rewritten.

For information, this has been done in #994.

As suggested, this PR has been split into different PRs: one for each notice. As of today, we do not have the statistics yet for GTFS datasets, but will update when we have them.